frontend/package.json

```diff
@@ -40,6 +48,7 @@
   "dependencies": {
     "@types/enzyme": "^3.10.3",
     "autobahn": "19.7.3",
+    "big.js": "^5.2.2",
```

Comment:
Can we `bn.js` instead of pulling another package (`big.js`)? (we already use `bn.js` in many other places)

Reply:
Sorry we cannot(pity), bn.js cannot show decimal but integer. @frol

Reply:
Do we really need decimal? We seem to only care about yoctoNEAR (integer) up until we display the values (millions of NEAR?)

Reply:
we display the decimal for Votes/Stake, it is now 0.17%, the bn.js will give 0 which is really weird...

Reply:
I would compute the percent value x100, then convert the result to regular JS Number (given it cannot be higher than 100%, the max x100 value would be 10000), divide by 100 and use toFixed(2).

Proceed at your own judgement here

Reply:
let me have a try